SwrveSDK/SDK/Talk/SwrveTriggerCondition.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+
+typedef enum SwrveTriggerOperator : NSUInteger {
+    SwrveTriggerOperatorAND,
+    SwrveTriggerOperatorOR,
+    SwrveTriggerOperatorEQUALS,
+    SwrveTriggerOperatorOTHER
```

Comment:
These are operator enums that are saved from the defined operator. If we ever expand on the logic in fulfilling the conditions then we can do it in a switch. :D